src/Explorer/Explorer.tsx

```diff
@@ -376,13 +366,59 @@ export default class Explorer {
       });
     }
 
-    useNotebook.getState().initializeNotebooksTree(this.notebookManager);
-
     this.refreshNotebookList();
 
     this._isInitializingNotebooks = false;
   }
 
+  public async allocateContainer(): Promise<void> {
+    const notebookServerInfo = useNotebook.getState().notebookServerInfo;
+    const isAllocating = useNotebook.getState().isAllocating;
+    if (notebookServerInfo && notebookServerInfo.notebookServerEndpoint === undefined && isAllocating === false) {
+      const provisionData = {
+        cosmosEndpoint: userContext.databaseAccount.properties.documentEndpoint,
+        dbAccountName: userContext.databaseAccount.name,
+        aadToken: userContext.authorizationToken,
```

Comment:
>aadToken [](http://example.com/codeflow?start=8&length=8)

nit; move account related info (sub, rg, account name, endpoint) together and move token up r down.